packages/gatsby-plugin-treat/tests/styling.test.js

```diff
@@ -1,8 +1,28 @@
+import {promisify} from 'util';
 import execa from 'execa';
 import * as path from 'path';
 import waitForLocalhost from 'wait-for-localhost';
+import treeKill from 'tree-kill';
 import getStyles from '../../../test-helpers/getStyles';
 import resolveBin from '../../../test-helpers/resolveBin';
+import { SIGKILL } from 'constants';
+
+const treeKillAsync = promisify(treeKill);
+
+const exec = (...args) => {
+  const childProcess = execa(...args);
+
+  childProcess.end = async () => {
```

Comment:
This is a custom function. You need to call it explicitly. Guess you could also override the built-in `kill`.